Gemfile

```diff
@@ -0,0 +1,74 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.1.2'
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem 'rails', '~> 7.0.4'
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem 'sprockets-rails'
+
+# Use Postgres as the database for Active Record
+gem 'pg'
+
+# Use the Puma web server [https://github.com/puma/puma]
+gem 'puma', '~> 5.0'
+
+gem 'rack-cors'
```

Comment:
- [**Optional**] React client could not still have access to your Rails API. You may kindly follow [these suggestions](https://stackoverflow.com/questions/58653710/cannot-fetch-data-from-rails-at-reactjs) to resolve the COR issue.

Reply:
implemented :+1: